Formula/ansible-lint.rb

```diff
@@ -124,6 +86,10 @@ class AnsibleLint < Formula
 
   def install
     virtualenv_install_with_resources
+    version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    site_packages = "libexec/lib/python#{version}/site-packages"
```

Comment:
```suggestion
    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
    site_packages = "libexec/lib/python#{xy}/site-packages"
```
Let's not shadow the variable `version` defined in the formula.

Reply:
Yup